node/config/types.go

```diff
@@ -158,6 +161,23 @@ type DealmakingConfig struct {
 	RetrievalPricing *RetrievalPricing
 }
 
+type IndexerProviderConfig struct {
+	config.Ingest
```

Comment:
Looking at the output changes in `doc_gen.go` looks like `cfgdocgen/gen.go` does not support embedded structs. 

Consider either fixing the generator or fork `config.Ingest` here?

Reply:
Probably want to copy config.Ingest into here so config comments get generated correctly